### ISSUE TITLE
A JavaScript error occured in the main process

### ISSUE THREAD

**Issue by Duffyboys** [dustinblackman/Championify#176]
Hey. When i try downloading the latest version it comes up with an error. Any idea on how to fix it? https://gyazo.com/0da4cebc5da4b71ae5ac2c04acf201f4

**Comment by dustinblackman** [comment]
No I've never seen that before. That looks like an error with Atom and your system. You've tried reinstalling?

**Comment by Duffyboys** [comment]
I tried to open the original one but that message come up. So, i tried to install it again and it said i need to abort other files. Those files all were Championify and there was about 7 of them stacked on top of each other. I had only downloaded Championify once in the past, mabye twice after i deleated the first one. There was an option to "abort and continue" whick was "recommened" or "ignor". I aborted and continued. The green loading bar came up and every few seconds an error message would come up. i tried downloading it again today and i got closer to getting it to work because it came up with the program with an error message. https://gyazo.com/b5f5b731b2afe850f4a7401e5206d694 . That is the error message. I am the admin of this computer and when i tried to click upload log, the button switch to failed.

Update: I think i got it. is this the version that is suppose to come up when i finish downloading the program? https://gyazo.com/107cdcbc9be0360d4d3bcbf4e800d1fe
